longestString/longestString.js

```diff
@@ -1,4 +1,14 @@
 /*
  * Write a function that accepts an array of strings.
  * Return the longest string in the array.
- */
\ No newline at end of file
+ */
+
+const longestString = (stringArray) => {
+	let currentLongest = stringArray[0];
+		for(strings of stringArray) {
```

Comment:
looking fancy with the for of loop! 
I recommend looking at code formatting. It is uncessecery to indent line 8. Doing so causes some readability issues, but your logic is sound and the code works.